examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+# Inference with ONNX Runtime
+import onnxruntime
+import time
+from torchvision import models, transforms as T
+import torch
+from PIL import Image
+import numpy as np
+import argparse
+import os
+import subprocess
+
+#Use most upto date weights
+inception_v3 = models.inception_v3(weights=models.Inception_V3_Weights.DEFAULT, progress=True)
+
+# Download ImageNet labels
+#!curl -o imagenet_classes.txt https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt
+
+def parse_input_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--fp16",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Perform fp16 quantization on the model before running inference',
+    )
+
+    parser.add_argument(
+        "--image_dir",
+        required=False,
+        default="../resnet50/images",
+        help='Target DIR for images to infer. Default is ./images'
+    )
+
+    parser.add_argument(
+        "--batch",
+        required=False,
+        default=1,
+        help='Batch size of images per inference',
+        type=int
+    )
+
+    parser.add_argument(
+        "--top",
+        required=False,
+        default=1,
+        help='Show top K of inference results',
+        type=int
+    )
+
+    parser.add_argument(
+        "--QPS",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show inference result in Queries-Per-Second QPS instead of inference duration (milliseconds)',
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show verbose output',
+    )
+    
+    return parser.parse_args()
+
+def get_image_list_in_dir(dir):
+    proc = subprocess.run("ls",
+                          shell=True,
+                          stdout=subprocess.PIPE,
+                          cwd=dir)
+    fileList = proc.stdout.decode().split('\n')
+    fileList
+    return fileList
+
+def softmax(x):
+    """Compute softmax values for each sets of scores in x."""
+    e_x = np.exp(x - np.max(x))
```

Comment:
subtraction with `np.max(x)` is it approximation to avoid overflows/underflows ?

Reply:
Thats what I thought too. I took this from an example when I did resnet50. If its an issue I can change